sky/backends/cloud_vm_ray_backend.py

```diff
@@ -200,8 +200,7 @@ def add_prologue(self,
         self.job_id = job_id
         # Should use 'auto' or 'ray://<internal_head_ip>:10001' rather than
         # 'ray://localhost:10001', or 'ray://127.0.0.1:10001', for public cloud.
-        # Otherwise, it will a bug of ray job failed to get the placement group
-        # in ray <= 2.4.0.
+        # Otherwise, it will a bug of ray job failed to get the placement group.
```

Comment:
nit: This sentence seems malformed - does it mean `Otherwise, ray will fail to get the placement group because of a bug in ray job`?